packages/editor-ui/src/router.ts

```diff
@@ -213,7 +213,7 @@ export const routes = [
 		},
 	},
 	{
-		path: '/workflows',
+		path: '/projects/:projectId/workflows',
```

Comment:
Should not this be backwards compatible?

Reply:
We just agreed we would then keep the old routes and they'll redirect the users to the proper new ones